seam/client.py

```diff
@@ -76,3 +79,34 @@ def _handle_error_response(self, response: requests.Response):
             raise SeamHttpInvalidInputError(error_details, status_code, request_id)
 
         raise SeamHttpApiError(error_details, status_code, request_id)
+
+    def _is_api_error_response(self, response: requests.Response) -> bool:
```

Comment:
If this doesn't use self it can be moved outside of the class and just be a normal internal function 
```suggestion
def is_api_error_response(response: requests.Response) -> bool:
```

Reply:
@razor-x what about using `@staticmethod` decorator instead?

Reply:
Similar, but `staticmethod` would really be for something you wanted to expose on the class. This is really just an internal private function.

Reply:
[`1ab722e` (#263)](https://github.com/seamapi/python/pull/263/commits/1ab722eadb2bc75cc5b66129258e6ad2bd6fd1e1)